### ISSUE TITLE
Unable to install cartopy

### ISSUE THREAD

**Issue by Jayayadav33** [SciTools/cartopy#2436]
I am using Ananconda on Windows 10. Python version: 3.11.7, I am unable to install it on my system. I have tried all the install as per the anaconda cartopy page: [](https://anaconda.org/conda-forge/cartopy) . I get the following error message:

**(PhD_Objective2) C:\Users\Admin>conda install cartopy
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.Collecting package metadata (repodata.json): done
Solving environment: unsuccessful initial attempt using frozen solve. Retrying with flexible solve.**

**Comment by akrherz** [comment]
Do you have `conda-forge` as your first channel?

**Comment by Jayayadav33** [comment]
I do not know if this is the first channel. I am kind of new to programming. But I had also tried this:
**conda config --add channels conda-forge
conda config --set channel_priority strict
conda install cartopy**
URL for this: https://github.com/conda-forge/cartopy-feedstock
But got the same result.

**Comment by dopplershift** [comment]
I would create a new environment and see if that works:

```
conda create -n myenv python=3.11
conda activate myenv
conda install cartopy
```

**Comment by lgolston** [comment]
Creating a fresh environment should solve this as suggested above. I would also check what version of conda you have as installed, as recent versions use libmamba under the hood and are less prone to this problem.

**Comment by Jayayadav33** [comment]
Yes. This is working in fresh environment. But not working in an existing environment. I am using anaconda navigator 2.5.0
